### PR TITLE
Implement keyboard shortcuts and combos

### PR DESCRIPTION
**Pull request by CtrlSpice**
This PR introduces a new custom hook called `useKeyCombo` to complement `useKeyPress` that will allow us to quickly create new keyboard shortcuts as the application grows.

It also defines the following shortcuts: 
-`r` to reload the page
-`Ctrl` + `l` to clear all traces
- `?` to show a modal with keyboard shortcuts and navigation tips (this will be implemented in the next PR)

Sorry, no video this time because it doesn't add much. 😄

### CODE REVIEW

File: desktopexporter/app/utils/use-key-press.ts [hunk 1]
```diff
@@ -33,3 +35,54 @@ export const useKeyPress = (targetKeys: string[]) => {
...
+              return event.metaKey;
+            case "Shift":
+              return event.shiftKey;
+            default:
+              return false;
+          }
+        })
+        .reduce((accumulator, currentValue) => {
+          return accumulator && currentValue;
+        });
```
Comment (jmorrell) [inline desktopexporter/app/utils/use-key-press.ts:61]: I see you getting fancy over there

**Review by jmorrell — APPROVED** [review]
I'm playing around with this locally, and OMG it's so much better to be able to cmd+tab to the window and not have to take my hands off the keyboard. This is such an improvement!

DALL·E 2023-05-14 19 11 28 - a corgi looking at a map at a fork in the road considering a shortcut, digital art

File: desktopexporter/app/utils/use-key-press.ts [hunk 2]
```diff
@@ -33,3 +35,54 @@ export const useKeyPress = (targetKeys: string[]) => {
   );
   return keyPressed;
 };
+
+export const useKeyCombo = (modifierKeys: string[], targetKeys: string[]) => {
```
Comment (jmorrell) [inline desktopexporter/app/utils/use-key-press.ts:39]: Just a note that you can use a more restricted enum type for `modifierKeys` here:

```typescript
type KeyString = "Alt" | "Control" | "Meta" | "Shift";
``` 

https://www.typescriptlang.org/play?#code/C4TwDgpgBA0hIGVgCcCWA7A5lAvFARAIIA2w+UAPgQMID26Ktx5V+AshMAIYsEIAWqAGZkA3AFgAUFKEBXdAGNgqelFkBnCHBB0AtgCNaACl20AJsNQRk29QC5Y8JGiwBtALoAaKN2SZOtg7qKBiYHgCUUADeUlBQxJxQyBDquFCmFkJWNvDqAHS6XGBGJubakTgAfNGxcXX16gDuqMAK-FClZuU1kvV9dQpcmgQkZHa1-f3JwLLI6FAAjBK9k-WDw-h0DMhM+OMrq-XTs-MATMuHA0PQ7Jw8+5dHnCdQAMwXj+s3AsJjE4-HOZQAAsH0uZggQi4slID0ecUB8yhxE0YMmAF9aujwh8FPR1EwIHliLRMEZkuocVJMdJJAB6OlQABytCg1h2yCkGi08D0hiMriIpHwXiggoAKiLwlIpAyoABRZAcgCEXM02j5xkFekK6DMIu8EqlQA

File: desktopexporter/app/utils/use-key-press.ts (hunk 1 above)
Comment (CtrlSpice) [reply desktopexporter/app/utils/use-key-press.ts:61]: 🎩 🧐

**Pull request merged**